apis/MessagesEndpoint.es6.js

```diff
@@ -0,0 +1,48 @@
+import Comment from '../models2/CommentModel';
+import Post from '../models2/PostModel';
+import Message from '../models2/MessageModel';
+import { runQuery, save } from '../apiBase/APIRequestUtils';
+
+const TYPE_MAP = {
+  't1': Comment,
+  't3': Post,
+  't4': Message,
+};
+
+const parseGetBody = (res, apiResponse, apiRequest) => {
+  const { body } = res;
+  body.data.children.forEach(c => apiResponse.addResult(TYPE_MAP[c.kind].fromJSON(c.data)));
+};
+
+const parsePostBody = (res, apiResponse, apiRequest) => {
+  const { body } = res;
+  console.log(body);
+};
+
+const path = (method, query={}) => {
```

Comment:
Could you change this to `getPath` or `path` for now and only handle the `GET` case? I've found that separating out the method paths has kept things a little cleaner.